.travis/check_docs.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+fatal=0
+
+function grep_check {
+  local pattern=$1
+  local description=$2
+  local fatalness=${3:-1}
+  x=$(grep -i -E -r -Hn "$pattern" --exclude="*.png" --exclude="*.svg" --exclude-dir="design_docs" --exclude-dir="contributing" documentation/)
+  if [ -n "$x" ]; then
+    echo "$description:"
+    echo "$x"
+    y=$(echo "$x" | wc -l)
+    ((fatal+=fatalness*y))
+  fi
+}
+
+# Check for latin abbrevs
+grep_check '[^[:alpha:]](e\.g\.|eg)[^[:alpha:]]' "Replace 'e.g'. with 'for example, '"
```

Comment:
The regexs fail to report errors if the problematic word is at line beginning (or line end).  The problem is masked if you happen to have already generated the HTML form. 

Separately the .travis directory seems like an odd location of the check_docs.sh, as the check is not bound to Travis.

Reply:
@k-wall Both script and location is just copied from https://github.com/strimzi/strimzi-kafka-operator/tree/master/.travis, so I hoped to minimize the changes from that

Reply:
@k-wall I can move the script to `scripts`. For the regexp... I'm inclined to say its ok that its not a catch-all, but a best effort thing. Unless you have quick solution in place to fix it :)

Reply:
It seems good enough.